.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-  - "6"
+  - "8"
```

Comment:
Would it make sense to move to 10 straight away? Would last longer...

Reply:
let's try.